lib/code_corps/stripe_service/stripe_connect_account.ex

```diff
@@ -4,28 +4,44 @@ defmodule CodeCorps.StripeService.StripeConnectAccountService do
 
   @api Application.get_env(:code_corps, :stripe)
 
+  @doc """
+  Used to create a remote `Stripe.Account` record as well as an associated local
+  `StripeConnectAccount` record.
+  """
   def create(attributes) do
     with {:ok, from_params} <- StripeConnectAccountAdapter.from_params(attributes),
          {:ok, %Stripe.Account{} = account} <- @api.Account.create(from_params),
          {:ok, params} <- StripeConnectAccountAdapter.to_params(account, attributes)
     do
-      %StripeConnectAccount{}
-      |> StripeConnectAccount.create_changeset(params)
-      |> Repo.insert
+      %StripeConnectAccount{} |> StripeConnectAccount.create_changeset(params) |> Repo.insert
     end
   end
 
+  @doc """
+  Used to update both the local `StripeConnectAccount` as well as the remote `Stripe.Account`,
+  using attributes sent by the client
+  """
   def update(%StripeConnectAccount{id_from_stripe: id_from_stripe} = account, %{} = attributes) do
     with {:ok, from_params} <- StripeConnectAccountAdapter.from_params(attributes),
          {:ok, %Stripe.Account{} = stripe_account} <- @api.Account.update(id_from_stripe, from_params),
-         {:ok, params} <- StripeConnectAccountAdapter.to_params(stripe_account, attributes),
-         {:ok, %StripeConnectAccount{} = updated_account} <- account |> StripeConnectAccount.webhook_update_changeset(params) |> Repo.update
+         {:ok, params} <- StripeConnectAccountAdapter.to_params(stripe_account, attributes)
     do
-      {:ok, updated_account}
+      account |> StripeConnectAccount.webhook_update_changeset(params) |> Repo.update
```

Comment:
This call will return `{:ok, updated_account}` anyway. No point in first calling it in the `with` block, then just passing it through in the `do` block.